cattle/compute/__init__.py

```diff
@@ -63,6 +47,33 @@ def instance_deactivate(self, req=None, instanceHostMap=None,
                                                         progress)
         )
 
+    def get_instance_host_from_map(self, instanceHostMap=None):
```

Comment:
Kind of a nitpick, but you shouldn't have `=None` here. when you default a param to None in python, you're implying that it is ok to not pass that parameter in, but that doesnt appear to be the intention here